src/components/descriptions/DescriptionsGenerator.ts

```diff
@@ -53,31 +59,32 @@ export async function generateDescriptions(
 /**
  * Calls the GPT API to generate the longer description.
  *
- * @param {string} data
- * @param {string} title
- * @param {number} average
- * @param {string} context
- * @param {string} key
- * @param {string} adjustedModel
- * @param {number} adjustedTemperature
- * @return {Promise<string>} Longer chart description.
+ * @return Longer chart description.
  */
-async function longerDescription(
-	data: string,
-	title: string,
-	average: number,
-	context: string,
-	key: string,
-	adjustedModel: string,
-	adjustedTemperature: number,
-): Promise<string> {
-	average = JSON.stringify(average);
+async function longerDescription({
+	data,
+	title,
+	average,
+	context,
+	key,
+	adjustedModel,
+	adjustedTemperature,
+}: {
+	data: string;
+	title: string;
+	average: number;
+	context: string;
+	key: string;
+	adjustedModel: string;
+	adjustedTemperature: number;
+}): Promise<string> {
+	const averageString = JSON.stringify(average);
 	const prompt =
```

Comment:
There are easier ways to do this.
Either with:
- simple, native string interpolation.
- Using the [template utility function on js-utilities](https://feedzai.github.io/js-utilities/docs/functions/string/template), which I think it would work out best for your needs.